.github/workflows/platformDeploy.yml

```diff
@@ -371,46 +371,13 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
-  postGithubComment:
```

Comment:
This diff looks odd, but `postGithubComment` was just moved to the bottom of the file without the Slack announcement steps in it.